README.md

```diff
@@ -58,4 +58,6 @@ yarn run
 
 This will display an interactive session where you can select the project and command you want.
 
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/grain-lang/grain)
```

Comment:
I think it makes a bit more sense to have this on the website's Getting Grain page than here, but I dunno what others might think.

Reply:
Why not in both places? Some people arrive here first.

Reply:
+1 on putting this on the Grain site, but I'm indifferent w.r.t. whether it should go here as well.

Reply:
I thought there was a difference between the gitpod docker files for "using Grain" and "developing Grain"? If so, we should be advertise "using Grain" on the website and "developing Grain" on the repo.

Reply:
The button launches the attached repo, so the develop button goes on the main Grain repo and gives you an environment to build the compiler in.  It can go on any page on the repo.

The develop with Grain is on the new template repo and just gives you the compiler binaries

Reply:
Makes sense. I think maybe we should not include a badge/image and instead just have a link under its own section.